### PR TITLE
Add vagrant setup

### PR DESCRIPTION
**Pull request by yitznewton**
I find Vagrant to be useful, so I put together an Ubuntu-based Vagrant setup for OpenCFP. I'm quite amenable to suggestions of changes, or closing the PR if it's not something y'all want to maintain as part of the project.

### CODE REVIEW

File: config/development.dist.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ cache:
 
 database:
   host: 127.0.0.1
-  database: cfp_travis
-  dsn: mysql:dbname=cfp_travis;host=127.0.0.1
+  database: cfp
+  dsn: mysql:dbname=cfp;host=127.0.0.1
```
Comment (yitznewton) [inline config/development.dist.yml]: Phinx specifies the dev database as `cfp`, hence this change

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  config.vm.box = "ubuntu/vivid64"
+
+  # config.vm.network "forwarded_port", guest: 80, host: 8080
+  config.vm.network "private_network", ip: "192.168.33.67"
```
Comment (yitznewton) [inline Vagrantfile]: I'm not sure of the best way to handle this, whether this IP should be committed, and how individuals are supposed to manage host network config

**Comment by mdwheele** [comment]
Hey @yitznewton! Thanks for the thought! We've had a few folks send development environments to help out and we think it's outside the scope of the project right now.

As an aside, for things like per-user Vagrant configuration, [this is what I do](https://github.com/mdwheele/vm/blob/master/Vagrantfile). I basically collapse per-user configuration from a YAML file on top of a default distributed configuration. Feel free to use it as an example for your own modification. The [Homestead](https://github.com/laravel/homestead) project is also a good example of building somewhat of a "usage framework" around vagrant.

If you have any questions, grab me here or on Twitter and maybe we can hop in IRC and discuss.

**Comment by yitznewton** [comment]
Sweet, thanks @mdwheele!

**Pull request closed without merging**